src/utils/boardIdUnserializer.ts

```diff
@@ -0,0 +1,6 @@
+import { makeMarshal } from '@endo/marshal';
```

Comment:
it's confusing to have `util/' and `utils/`. It's also annoying for path completion.

`config.ts` is for the whole app and not a utility, so just move that to `src/`.

Reply:
Ah yea I probably forgot util already existed. Just moved to src.